.eslintrc.js

```diff
@@ -1,5 +1,8 @@
 module.exports = {
   extends: ['@energyweb'],
+  rules: {
+    "@typescript-eslint/no-shadow": ["warn", {"ignoreFunctionTypeParameterNameValueShadow": true}],
```

Comment:
Is this rule allows to shadow function arguments by local variables? If so I think it is largely breaks the purpose of the rule